repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -299,6 +304,21 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       }
     }
 
+    def doUserTokensUpdate(userId: String, authToken: String, ttl: Int): Unit = {
+      if (CacheInvalidationFeedListener.invalidationEnabled()) {
```

Comment:
I made it conditional the more the actual update code grew. It is probably premature optimization, but the other code was so ugly.